Libraries/NativeModules/specs/NativeDialogManagerAndroid.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+import Platform from '../../Utilities/Platform';
+
+type DialogAction = 'buttonClicked' | 'dismissed';
+type DialogButtonKey = 'buttonPositive' | 'buttonNegative' | 'buttonNeutral';
+type DialogOptions = {|
+  +title?: string,
+  +message?: string,
+  +buttonPositive?: string,
+  +buttonNegative?: string,
+  +buttonNeutral?: string,
+  +items?: Array<string>,
+  +cancelable?: boolean,
+|};
+
+export interface Spec extends TurboModule {
+  +getConstants: () => {|
+    buttonClicked: DialogAction,
+    dismissed: DialogAction,
```

Comment:
Unfortunately, the Codegen system doesn't support unions. 😔

 So, I think it's best if we change this to `string`.

Reply:
What about keeping this as it is and adding and updating `DialogAction` to `type DialogAction = string`?